tools/jdk/BUILD

```diff
@@ -333,3 +333,16 @@ test_suite(
     tests = [":windows_tests"],
     visibility = ["//tools:__pkg__"],
 )
+
+# Replacement for the embedded JDK as host javabase.
+alias(
+    name = "remotejdk",
+    actual = select({
+        "//src/conditions:darwin": "@remotejdk_macos//:jdk",
+        "//src/conditions:darwin_x86_64": "@remotejdk_macos//:jdk",
+        "//src/conditions:windows": "@remotejdk_win//:jdk",
+        "//src/conditions:linux_aarch64": "@remotejdk_linux_aarch64//:jdk",
+        "//conditions:default": "@remotejdk_linux//:jdk",
```

Comment:
Why is that? I want to hide the embedded JDK without breaking anyone. This is basically maintaining the status quo where people don't have to specify a host_javabase.

Reply:
Are you referring to this status quo?
https://github.com/bazelbuild/bazel/blob/1e68ace8d32e94e099cf11d2e34c76d844a47dbc/src/BUILD#L210

That select is used when building Bazel itself, and binaries with embedded JDKs are provided for a fixed set of platforms.

This select will be used by Bazel installs on arbitrary platforms, and defaulting to linux on non-linux platforms is unhelpful. It's a regression compared to the current behaviour of the binaries without an embedded JDK, which I think default to using the local JDK as the server, host, and target javabases.

Reply:
I see what you mean now - I've to do some research and will come back to you.

Reply:
I was about to say the same :) I think it's fine to submit "are we on `$OPERATING_SYSTEM` to `@bazel_tools` somewhere. We already have `//tools/osx:darwin`, which should probably stay as an alias to whatever you come up with.

/cc @ilya-biryukov @mhlopko @katre  because this is also interesting for C++ and platforms in general.

Reply:
re: unknown platforms, I also prefer erroring out. It only happens with the incompatible flag set so it should give people heads-up, and I'd expect that those OSes are a niche use case which is served well enough by an appropriate `--host_javabase` flag.

Reply:
As discussed offline: that already exists in @bazel_tools and I'm using it (I had to add linux as an option though).